docs/source/get_started.rst

```diff
@@ -0,0 +1,192 @@
+===========
+Get started
+===========
+
+Installation
+============
+To download and install the library from pypi simply execute:
+
+``pip install pytorch-datastream``
+
+Usage
+=====
+
+Dataset from subscriptable
+--------------------------
+Simple usage with ``Dataset.from_subscriptable``. This is mostly useful for
+simple examples. It is often preferable to use ``Dataset.from_dataframe``.
+
+.. highlight:: python
+.. code-block:: python
+
+    from datastream import Dataset
+
+    fruits_and_cost = (
+        ('apple', 5),
+        ('pear', 7),
+        ('banana', 14),
+        ('kiwi', 100),
+    )
+
+    dataset = (
+        Dataset.from_subscriptable(fruits_and_cost)
+        .map(lambda fruit, cost: (
+            fruit,
+            cost * 2,
+        ))
+    )
+
+    print(dataset[2]) # ('banana', 28)
+
+Dataset from pandas dataframe
+-----------------------------
+This example tries to show a simple data pipeline in pseudo-code where a
+dataset is is created from a dataframe, then images are read from disk,
+augmented, and preprocessed before training.
```

Comment:
Requires PIL and imgaug to be installed

Reply:
I'm guessing you think that is bad? We cannot have a self-contained example and show a possible pipeline at the same time

Reply:
no, its totally fine, just have a small note on it (example requires PIL and imgaug to be installed)